apps/bmd/src/AppCardlessVoting.test.tsx

```diff
@@ -16,11 +17,7 @@ import {
 
 import withMarkup from '../test/helpers/withMarkup'
 
-import {
-  adminCardForElection,
-  advanceTimersAndPromises,
-  pollWorkerCardForElection,
-} from '../test/helpers/smartcards'
+import { advanceTimersAndPromises } from '../test/helpers/smartcards'
```

Comment:
while you are updating the smartcard things FYI that I also put a version of this functions in the test-utils lib https://github.com/votingworks/vxsuite/blob/main/libs/test-utils/src/advanceTimers.ts

Reply:
Yeah, I saw that. I remembered this one as being slightly incompatible so I didn't update it.